common/src/main/java/com/viaversion/viabackwards/protocol/protocol1_16_4to1_17/packets/BlockItemPackets1_17.java

```diff
@@ -147,6 +148,8 @@ public void registerMap() {
                     short slot = wrapper.passthrough(Type.SHORT);
 
                     Item carried = wrapper.read(Type.FLAT_VAR_INT_ITEM);
+                    wrapper.write(Type.FLAT_VAR_INT_ITEM, handleItemToClient(carried));
```

Comment:
I changed my comment before. You should be tracking (copied) the item before remapping it to 1.16.4

Reply:
Makes sense, since we aren't mapping it back to the server item after it's saved